config/home/clusterworkspacetype-use-clusterrole.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:kcp:universal-clusterworkspacetype-use
+rules:
+- apiGroups: ["tenancy.kcp.dev"]
+  resources: ["clusterworkspacetypes"]
+  resourceNames: ["universal"]
+  verbs: ["use"]
```

Comment:
don't think that's how it works now. No roles+bindings necessary anymore deep in the hierarchy, only in root.